src/operations/brightness.jl

```diff
@@ -0,0 +1,101 @@
+import ImageCore: clamp01
+import Statistics: mean
+
+"""
+    AdjustContrastBrightness <: ImageOperation
+
+Description
+--------------
+
+Adjusts the brightness and contrast of each pixel.
```

Comment:
It is more self-explaining if you can add the math `clamp(α * img[i] + β * M)` here.